packages/frontend-core/src/api/automations.js

```diff
@@ -26,9 +26,13 @@ export const buildAutomationEndpoints = API => ({
   /**
    * Gets a list of all automations.
    */
-  getAutomations: async () => {
+  getAutomations: async ({ enrich }) => {
+    let url = "/api/automations?"
+    if (enrich) {
+      url += "enrich=true"
```

Comment:
Can we use `URLSearchParams` here to add this to the URL? Helpful if this ever needs to be extended with other parameters.
e.g.
```
const params = new URLSearchParams()
if (enrich) {
  params.set("enrich", true)
}
const url = `/api/automations?${params.toString()}`
```